package.json

```
@@ -56,5 +60,17 @@
                 "path": "./snippets/monkeyc.json"
             }
         ]
+    },
+    "scripts": {
+        "vscode:prepublish": "npm run compile",
+        "compile": "tsc",
+        "watch": "tsc -watch"
+    },
+    "devDependencies": {
+        "@types/node": "^14.0.1",
+        "vscode": "^1.1.37"
+    },
```

Please add typescript dev dependency.

```suggestion
        "tslint": "^5.12.1",
        "typescript": "^3.3.3",
        "vscode": "^1.1.37"
    },
```